Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -1168,6 +1205,188 @@ function Set-IdentityUri {
+                                },
+                                @{
+                                    Name =  "MetadataContext-Deployment-Deployment Connection String";
+                                    Value = $metaDataConnectionString
+                                },
+                                @{
+                                    Name = "MetadataContext-Web.config Connection String";
+                                    Value = $metaDataConnectionString
+                                }
+                                ,
+                                @{
+                                    Name = "Discovery Service Endpoint";
+                                    Value = $discoveryServiceUrl
+                                }
+                                ,
+                                @{
+                                    Name = "Use Discovery Service";
+                                    Value = $noDiscoveryService
+                                }
+                                ,
+                                @{
+                                    Name = "Current Domain";
+                                    Value = $commonConfig.webServerDomain
+                                }
+                                ,
+                                @{
+                                    Name = "Identity Provider Search Service Api Secret";
+                                    Value = $registrationApiSecret
+                                }
+                            )
+    
+    return $webDeployParameters
+}
+
+function Confirm-ServiceConfig{
+    param(
+        [Parameter(Mandatory=$true)]
+        [Hashtable] $commonConfig,
+        [Parameter(Mandatory=$true)]
+        [Hashtable] $serviceConfig
+    )
+
+    Confirm-SettingIsNotNull -settingName "common.sqlServerAddress" -settingValue $commonConfig.sqlServerAddress
+    Confirm-SettingIsNotNull -settingName "common.metadataDbName" -settingValue $commonConfig.metadataDbName
+    Confirm-SettingIsNotNull -settingName "common.webServerDomain" -settingValue $commonConfig.webServerDomain
+    Confirm-SettingIsNotNull -settingName "common.clientEnvironment" -settingValue $commonConfig.clientEnvironment
+    Confirm-SettingIsNotNull -settingName "$serviceConfig.appName" -settingValue $serviceConfig.appName
+    Confirm-SettingIsNotNull -settingName "$serviceConfig.appPoolName" -settingValue $serviceConfig.appPoolName
+    Confirm-SettingIsNotNull -settingName "$serviceConfig.siteName" -settingValue $serviceConfig.siteName
+}
+
+function Confirm-SettingIsNotNull{
+    param(
+        [Parameter(Mandatory=$true)]
+        [string] $settingName,
+        [string] $settingValue
+    )
+
+    if([string]::IsNullOrEmpty($settingValue)){
+        Write-DosMessage -Level "Fatal" -Message "You must specify a valid $settingName in config."
+    }
+}
+
+function Get-MetadataConnectionString{
```

Comment:
I think we already have a function that does this: https://github.com/HealthCatalyst/Fabric.Identity/pull/299/files#diff-8cbfa091db649c4bbaf53cade86637cdR307

Reply:
I didn't add a new function for Get-MetadataConnectionString, there is still only one, I think it must just be a parenthesis change or something.

Reply:
What is the difference between the function on line 307 vs the function on line 1310?

The one on line 307 is called `Get-MetadataDatabaseConnectionString` the one on line 1310 is called `Get-MetadataConnectionString`